src/MainWindow.cpp

```diff
@@ -247,6 +248,23 @@ MainWindow::MainWindow(QWidget* parent):
 
     // start centered
     move(QApplication::desktop()->screen()->rect().center() - rect().center());
+
```

Comment:
should all of this really be done at the end of the ctor? Could you please improve the variable names? `mGenMenu` is not even a member and it starts with `m`.

Reply:
I basically repeated what I have seen in the `ResultsViewer` constructor. I don't know where else to put it. It seems that the pattern here is that the appearance of the form is fully determined after the ctor is executed, so from this POV, it makes sense to group the stuff that defines UI in there.